tests/reference/asr-func_04-eef2656.stdout

```diff
@@ -0,0 +1,82 @@
+(TranslationUnit
```

Comment:
This reference output should not be generated. It will go away once you revert changes at https://github.com/lcompilers/lpython/pull/2659/files#r1576400631.

Reply:
This reference output should be removed.